templates/main.html

```diff
@@ -7,7 +7,7 @@
     <meta property="og:title" content="GCI Current Leaders">
     <meta property="og:description" content="Google Code-in 2017 Current Leaders">
     <meta property="og:image" content="{{{rootURL}}}/static/images/GCI-logo.jpg">
-    <link rel="stylesheet" href="static/css/main.css">
+    <link rel="stylesheet" href="../static/css/main.css">
```

Comment:
Why?

Reply:
That was leftover from the initial folder restructuring I proposed, I can remove it if you'd like.

Reply:
https://deploy-preview-52--gci-leaders.netlify.com/static/css/main.css < Does exist, so revert it.